boards/PDM/Src/Io/Io_Efuse_Aux1Aux2.c

```diff
@@ -0,0 +1,131 @@
+#include "Io_Efuse_Aux1Aux2.h"
+#include "configs/Io_EfuseConfig.h"
+
+static struct Efuse_Context *aux1_aux2_efuse;
+
+void Io_Efuse_Init_Aux1Aux2(SPI_HandleTypeDef *const hspi)
+{
+    assert(hspi != NULL);
+
+    aux1_aux2_efuse = Io_Efuse_Create(
+        hspi, CSB_AUX1_AUX2_GPIO_Port, CSB_AUX1_AUX2_Pin,
+        FSOB_AUX1_AUX2_GPIO_Port, FSOB_AUX1_AUX2_Pin, FSB_AUX1_AUX2_GPIO_Port,
+        FSB_AUX1_AUX2_Pin, PIN_AUX1_GPIO_Port, PIN_AUX1_Pin, PIN_AUX2_GPIO_Port,
+        PIN_AUX2_Pin);
+}
+
+void Io_Efuse_EnableAux1(void)
+{
+    HAL_GPIO_WritePin(PIN_AUX1_GPIO_Port, PIN_AUX1_Pin, GPIO_PIN_SET);
```

Comment:
This should be in Io_Efuse.c. This also applies to the couple of functions below.

Reply:
Sure, I can make that change. I hadn't originally because it was such a simple  function

Reply:
This was changed